exchange_server/tests/test_e2e.py

```diff
@@ -0,0 +1,32 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import pytest
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.dev.testing import requires_py3
+from datadog_checks.exchange_server.metrics import METRICS_CONFIG
+
+
+def get_exchange_server_metrics():
```

Comment:
wow, this is complex. Maybe it would have been easier just asserting that all the metrics emitted are the 6 known ones

Reply:
After talking with @ofek about possible solutions, we agreed that this was the best and most complete. It is about ensuring that the metrics corresponding to MsExchange are not collected (those whose name begins like this)